flet/PyAlgoMate.py

```diff
@@ -39,21 +41,27 @@
 
 logging.getLogger("requests").setLevel(logging.WARNING)
 
+
 def GetFeedNStrategies(creds):
     with open("strategies.yaml", "r") as file:
         config = yaml.safe_load(file)
 
+    sentry_dns = creds.get('SENTRY', {}).get('SENTRY_DNS')
```

Comment:
Can you move this to the place where papertrail and telegram are being read as GetFeedNStrategies has nothing to do with third party integrations